docs/cloud-native-security/vuln-management-faq.asciidoc

```diff
@@ -11,6 +11,10 @@ The CNVM integration uses various security data sources. The complete list can b
 
 CNVM uses the open source scanner https://github.com/aquasecurity/trivy[Trivy] v0.35.
 
+*What system architectures are supported?*
+
+Because of Trivy's limitations, CNVM works only on ARM. Other architectures are not supported.
```

Comment:
```suggestion
Because of Trivy's limitations, CNVM can only be deployed on ARM-based VMs. However, it can scan hosts regardless of system architecture.
```